hutch_python/tests/conf.yaml

```diff
@@ -0,0 +1,48 @@
+hutch: mfx
+
+daq:
+  lcls: 1
+  platform: 4
+
+happi:
+  beamline:
+    - FEE
+    - HXD
+    - MFX
+
+lightpath: MFX
```

Comment:
Lets let lightpath figure out the rest of the beamline? Seems redundant to have to the beamlines, hutch and lightpath specified

Reply:
I agree that this particular `conf.yaml` is very redundant. Is it still reasonable to allow us to specify other `happi` imports in the configuration file? I may end up wrapping the `lightpath` Plugin into the `hutch` Plugin.

Reply:
That makes sense. I think standard deployment should be able to load everything just from hutch name. But you are correct, fancier things should be allowed.